src/java.desktop/windows/classes/sun/awt/windows/WPathGraphics.java

```diff
@@ -95,11 +95,8 @@ final class WPathGraphics extends PathGraphics {
     private static boolean useGDITextLayout = true;
     private static boolean preferGDITextLayout = false;
     static {
-        @SuppressWarnings("removal")
-        String textLayoutStr =
-            java.security.AccessController.doPrivileged(
-                   new sun.security.action.GetPropertyAction(
-                         "sun.java2d.print.enableGDITextLayout"));
+        String textLayoutStr = System.getProperty(
+                     "sun.java2d.print.enableGDITextLayout");
 
         if (textLayoutStr != null) {
             useGDITextLayout = Boolean.getBoolean(textLayoutStr);
```

Comment:
This line seems wrong: `textLayoutStr` is a value of the property, and we now read a new property with the value of the `sun.java2d.print.enableGDITextLayout` property.

Is it what's intended? The following line—`textLayoutStr.equalsIgnoreCase("prefer")`—assumes it's not the intended usage.

Reply:
If there's an existing problem, let's examine it under a separate bug. This change is about equivalence

Reply:
Yes, it's an existing problem, I'm just bringing it up.